interval.py

```diff
@@ -177,3 +174,12 @@ def none(pixels, args):
     for y in range(len(pixels)):
         intervals.append([len(pixels[y])])
     return intervals
+
+# Helper functions
+
+def load_interval_file(args):
```

Comment:
I also took the liberty to make the interval file loading uniform between `file` and `file-edges`.